scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -434,16 +434,14 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
      * [scanned path][scanPath].
      */
     protected fun relativizePath(scanPath: File, scannedFilename: File): String {
-        val relativePathToScannedFile = if (scanPath.isFile) {
-            if (scanPath.isAbsolute) {
-                // Scanners are called with the current working directory unchanged. To avoid absolute paths, make a
-                // path to a file relative to the current working directory in the absence of a scan root directory.
-                scanPath.relativeTo(File(".").absoluteFile)
+        val relativePathToScannedFile = if (scannedFilename.isAbsolute) {
```

Comment:
commit-msg: cannot see the added comments, I assume they went into a different commit meanwhile?

Reply:
They are in https://github.com/heremaps/oss-review-toolkit/pull/2327/commits/918ec04558596bdeb0091be0308458385eb57922 (but that SHA1 will change with my next force-push).